### PR TITLE
bsp: Restructuring BSP folders.

### PR DESCRIPTION
**Pull request by espzav**
# ESP-BSP Pull Request checklist

> Note: For new BSPs create a PR with this [link](https://github.com/espressif/esp-bsp/compare/main...my-branch?quick_pull=1&template=pr_template_bsp.md).

- [x] Version of modified component bumped
- [x] CI passing

# Change description
Moved all BSPs into `bsp` folder. All paths was changed (component url and version in BSPs was changed too)

### CODE REVIEW

**Comment by espzav** [comment]
**PLEASE, MERGE THIS PULL REQUEST BEFORE ANY OTHER CHANGES IN BSP!!!!**

**Comment by espzav** [comment]
@tore-espressif PTAL

File: bsp/esp-box-3/idf_component.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
-version: "1.1.1"
+version: "1.1.1~1"
```
Comment (tore-espressif) [inline bsp/esp-box-3/idf_component.yml:2]: To all boards: It is not necessary to bump the version now.

The examples in already uploade versions work fine. And this restructuralization won't affect them

Comment (espzav) [reply bsp/esp-box-3/idf_component.yml:2]: The link to component was changed and when you click to link in Component Registry, the link will be bad.

Comment (tore-espressif) [reply bsp/esp-box-3/idf_component.yml:2]: hmm, that is right

**Review by tore-espressif — APPROVED** [review]
Thanks @espzav !

**Pull request merged**